unity-renderer/Assets/DCLPlugins/MapTexturePlugin/MapTexturePlugin.cs

```diff
@@ -52,7 +47,7 @@ private static async UniTaskVoid DownloadTexture(string url, BaseVariable<Textur
                 }
                 else
                 {
-                    Texture2D tex = new Texture2D(0, 0, TextureFormat.ETC2_RGBA8, false, true);
+                    Texture2D tex = new Texture2D(0, 0, GraphicsFormatUtility.GetGraphicsFormat(TextureFormat.RGBA32, false), TextureCreationFlags.None);
```

Comment:
Could not we just use `textureFormat` instead? It was the only oversight here I believe?

Reply:
What I dont like about that approach is that we rely on a fixed set of textures formats and it doesnt scale that much. With this one we just ask the system to return a compatible GraphicsFormat with the needs we have.

Reply:
Can we delete `PRIORITIZED_FORMATS` then?